quickstarts/microsoft.documentdb/cosmosdb-sql-rbac/azuredeploy.json

```diff
@@ -0,0 +1,115 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "metadata": {
+    "_generator": {
+      "name": "bicep",
+      "version": "0.3.539.46024",
+      "templateHash": "10508405959747151395"
+    }
+  },
+  "parameters": {
+    "location": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description": "Location for all resources."
+      }
+    },
+    "accountName": {
+      "type": "string",
+      "defaultValue": "[format('sql-rbac-{0}', uniqueString(resourceGroup().id))]",
+      "metadata": {
+        "description": "Cosmos DB account name, max length 44 characters"
+      }
+    },
+    "roleDefinitionName": {
+      "type": "string",
+      "defaultValue": "My Read Write Role",
+      "metadata": {
+        "description": "Friendly name for the SQL Role Definition"
+      }
+    },
+    "dataActions": {
+      "type": "array",
+      "defaultValue": [
+        "Microsoft.DocumentDB/databaseAccounts/readMetadata",
+        "Microsoft.DocumentDB/databaseAccounts/sqlDatabases/containers/items/*"
+      ],
+      "metadata": {
+        "description": "Data actions permitted by the Role Definition"
+      }
+    },
+    "principalId": {
+      "type": "string",
+      "metadata": {
+        "description": "Object ID of the AAD identity. Must be a GUID."
+      }
+    }
+  },
+  "functions": [],
+  "variables": {
+    "accountName_var": "[toLower(parameters('accountName'))]",
```

Comment:
remove the decompile fluff...  i.e. any _var, _param, _resource appendages

Reply:
Fixed in next commit